modules/storages/app/forms/storages/admin/storages/add_projects_autocompleter_form.rb

```diff
@@ -38,6 +38,7 @@ class AddProjectsAutocompleterForm < ApplicationForm
               visually_hide_label: true,
               validation_message: project_ids_error_message,
               autocomplete_options: {
+                with_search_icon: true,
```

Comment:
@akabiru I tried to find another way to get the option in there but miserably failed at that :see_no_evil: 
After I have been looking around, I am also not so sure we want to introduce a new way for adding options. I am not too happy about this approach but still see it as the least intrusive way.